tools/scripts/build/mobile-curriculum.test.ts

```diff
@@ -27,22 +33,36 @@ if (envData.clientLocale == 'english' && !envData.showUpcomingChanges) {
     });
 
     test('the files generated should have the correct schema', () => {
-      const fileArray = fs.readdirSync(`${mobileStaticPath}/mobile`);
+      const fileArray = fs.readdirSync(
+        `${mobileStaticPath}/curriculum-data/${ver}`
+      );
 
       fileArray
         .filter(fileInArray => fileInArray !== 'availableSuperblocks.json')
         .forEach(fileInArray => {
           const fileContent = fs.readFileSync(
-            `${mobileStaticPath}/mobile/${fileInArray}`
+            `${mobileStaticPath}/curriculum-data/${ver}/${fileInArray}`,
+            'utf-8'
           );
 
           const result = validateMobileSuperBlock(JSON.parse(fileContent));
 
           if (result.error) {
-            throw new AssertionError(result.error, `file: ${fileInArray}`);
+            throw new AssertionError(
+              result.error.toString(),
+              `file: ${fileInArray}`
+            );
           }
         });
     });
+
+    test('All SuperBlocks should be present in the mobile SuperBlock object', () => {
+      Object.values(SuperBlocks).forEach(superBlockKey =>
+        expect(
+          Object.keys(superBlockMobileAppOrder).includes(superBlockKey)
+        ).toBe(true)
+      );
+    });
```

Comment:
```suggestion
    test('All SuperBlocks should be present in the mobile SuperBlock object', () => {
      expect(Object.keys(superBlockMobileAppOrder)).toEqual(
        expect.arrayContaining(Object.values(SuperBlocks))
      );
      expect(Object.keys(superBlockMobileAppOrder)).toHaveLength(
        Object.values(SuperBlocks).length
      );
    });
```
`expect(booleanThing).toBe(true)` is often a bit opaque because the output is typically equivalent to "Something went wrong... :shrug:".  If you ask Jest to compare objects or arrays directly, it'll give you a diff if it fails. Then it's easy to see why it's failing.